zaza/openstack/charm_tests/keystone/tests.py

```diff
@@ -150,6 +151,97 @@ def test_key_distribution_and_rotation(self):
                              .format(pprint.pformat(unit_repo),
                                      pprint.pformat(lead_repo)))
 
+    def test_rotate_admin_password(self):
```

Comment:
is it intentional to backport this test?, because there is a piece of code in charm-keystone missing.

```
2023-05-18 17:25:58.468921 | focal-medium | 2023-05-18 17:25:58 [INFO] juju.errors.JujuError: ['action "rotate-admin-password" not defined on unit "keystone/2"']
```